src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+
+  if (days <= 2) {
+    return days * price;
+  }
```

Comment:
Instead of checking if 'days' is less than or equal to 2, you can simply start with checking the larger conditions first and return early. This way, you don't need to check for 'days <= 2' explicitly, as it will be the default case if the other conditions aren't met.